frontend/packages/kubevirt-plugin/integration-tests/tests/vm.cloud-init.scenario.ts

```diff
@@ -0,0 +1,62 @@
+import { testName } from '../../../../integration-tests/protractor.conf';
```

Comment:
Could you use '@console/internal-integration-tests/protractor.conf'

Reply:
done